app/views/users/_blocked_person.haml

```diff
@@ -0,0 +1,12 @@
+.media.blocked_person{:id => person.id}
+  .pull-right
+    = link_to t('users.privacy_settings.stop_ignoring'), block_path(block),  class: 'btn btn-danger', method: :delete
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.